src/dropdown/dropdown.ts

```diff
@@ -83,16 +86,26 @@ export class NgbDropdown {
   @Input('open') _open = false;
 
   /**
+   * Placement of a dropdown. Use "top-right" for dropups.
```

Comment:
Again, I think you should either list the acceptable values in the doc or type it appropriately